codecs/README.md

```diff
@@ -0,0 +1,32 @@
+# Codecs
+
+This folder contains a self-contained sub-project for each codec that squoosh supports.
+
+## Build
+
+Each subproject can be built using the following commands:
+
+```
+$ npm install
+$ npm run build
+```
+
+This will build two files: `<codec name>_codec.js` and `<codec name>_codec.wasm`. Due to some current limitations in Emscripten, both of these files have to be served from the root folder. When the `.js` file is loaded, a global `<codec name>` is created with the same API as an [Emscripten `Module`](https://kripken.github.io/emscripten-site/docs/api_reference/module.html).
```

Comment:
"folder"? As in, the root of the server? Is there an emscripten ticket to remove this limitation?

Reply:
Seems a shame that it creates a global. Is there an emscripten ticket for creating modules?

Reply:
Not sure, I mentioned it to the wasm people on the feasibility study, but haven’t followed up. Basically, the path that you use for output is hard-coded in the the JS file >.>

Reply:
Seems like `Module["locateFile"]` is the answer https://kripken.github.io/emscripten-site/docs/api_reference/module.html#affecting-execution

Reply:
Cool. Missed that. I’ll remove the remark.